tests/Plugins/ScientificSoftware_EndToEndTest.py

```diff
@@ -28,7 +28,10 @@ def test_NoCitation(self, pat_args, snapshot):
         """Test if no CITATION file exists in the repository."""
         result = CliRunner().invoke(
             app,
-            pat_args,
+            pat_args
```

Comment:
I don't think that we need to explicitly provide this argument anymore, as the default should be that we validate that a citation file exists.